lte/gateway/docker/docker-compose.yaml

```diff
@@ -169,15 +152,16 @@ services:
     container_name: ctraced
     healthcheck:
       test: ["CMD", "nc", "-zv", "localhost", "50079"]
-      timeout: "4s"
-      retries: 3
     command: /usr/bin/env python3 -m magma.ctraced.main
 
   sctpd:
     <<: *ltecservice
     container_name: sctpd
     ulimits:
       core: -1
+    healthcheck:
+      test: ["CMD", "bash", "-c", "[ -S /tmp/sctpd_downstream.sock ]"]
```

Comment:
nitpick: We could also write `test: "test -S /tmp/sctpd_downstream.sock"`. Personally I find that more readable, especially the combination of square brackets for Yaml lists and for the bash `test` command are a bit hard to parse.

Reply:
And as a compromise if we want to stay consistent with the `["CMD", ...]` syntax used in this file:
```
test: ["CMD", "test", "-S", "/tmp/sctpd_downstream.sock"]
```
disclaimer: This one I didn't test, but I think it should work.